go/obscuronode/enclave/db/storage.go

```diff
@@ -266,6 +261,21 @@ func (s *storageImpl) GetTransaction(txHash common.Hash) (*types.Transaction, co
 	return tx, blockHash, blockNumber, index, nil
 }
 
+func (s *storageImpl) GetSender(txHash common.Hash) (common.Address, error) {
+	tx, _, _, _, err := s.GetTransaction(txHash) //nolint:dogsled
```

Comment:
`dogsled` is an amazing name for those dropped variables haha. Does GetTransaction need to return all those other things? Goland suggested that nothing else was calling the function (but it's lied to me before).

Reply:
we'll probably use them as we implement more rpc endpoints.
Those params are taken from geth, so I assumed we'd need them